tests/templates/test_tensornetworks/test_MPS.py

```diff
@@ -0,0 +1,102 @@
+from typing import DefaultDict
+import pytest
+import sys
+sys.path.append("../")
```

Comment:
```suggestion
import pytest
```
I don't think we need these.